docs/gallery.md

```diff
@@ -0,0 +1,48 @@
+# Gallery
+
+This page shows examples of figures that can be visualized in CARET.
+CARET visualizes these figures on jupyter-notebook.
+
+## Message flow
+
+![message_flow](./imgs/message_flow_sample.png)<br>
+Source code : [Message flow](https://github.com/tier4/CARET_analyze/blob/main/src/caret_analyze/plot/bokeh/message_flow.py)
```

Comment:
I think sample-code or related-visualization-page are needed instead of link to Source code.